molecule/snap/cleanup.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Prepare molecule instance
+  hosts: all[0]
+  gather_facts: no
+  tasks:
+  - name: Create a network with custom IPAM config
+    delegate_to: localhost
+    community.general.docker_network:
```

Comment:
Would you mind using docker_network instead of `community.general.` ? 
While you're right in the fact it's the way forward, I believe it has to stay until I phase 2.9 out.

I also want to update everything in one go.